### PR TITLE
[251] Add Feedback Message

### PR DESCRIPTION
**Pull request by thisoldbear**
- Add textarea component.
- Conditionally render form field depending on user permission.

### CODE REVIEW

File: mresponse/frontend/app/src/components/textarea/textarea.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      width: calc(100% - 20px);
+      padding: 10px;
+      background-color: $off-white;
+      border-radius: 3px;
+      border: 0.5px solid $border-color;
+      font-size: 14px;
+      line-height: 1.6;
+
+      &::placeholder {
+          color: rgba(#38383d, 0.3);
```
Comment (thisoldbear) [inline mresponse/frontend/app/src/components/textarea/textarea.scss]: Waiting on Matt C at Mozilla for the correct colors

File: mresponse/frontend/app/src/components/textarea/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      placeholder,
+      value,
+      onChange,
+      disabled,
+      rows = 3,
+      maxLength
+    } = this.props
+    return (
+      <textarea
+        placeholder={placeholder || 'Add your response'}
```
Comment (tm-kn) [inline mresponse/frontend/app/src/components/textarea/index.js]: Why do we want a default placeholder on a generic component? I'd keep it empty.

File: mresponse/frontend/app/src/components/textarea/index.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+export default class Textarea extends React.Component {
+  render () {
+    const {
+      placeholder,
+      value,
+      onChange,
+      disabled,
+      rows = 3,
+      maxLength
```
Comment (tm-kn) [inline mresponse/frontend/app/src/components/textarea/index.js]: Do we want `...rest` to include any not-predicted props?

File: mresponse/frontend/app/src/components/textarea/index.js (hunk 2 above)
Comment (thisoldbear) [reply mresponse/frontend/app/src/components/textarea/index.js]: @tm-kn I adapted the component from`components/input-field/index.js`, which has a default placeholder. Happy to remove.

File: mresponse/frontend/app/src/components/textarea/index.js (hunk 3 above)
Comment (thisoldbear) [reply mresponse/frontend/app/src/components/textarea/index.js]: True 👌

File: mresponse/frontend/app/src/components/input-field/field.scss [hunk 4]
```diff
@@ -3,10 +3,10 @@
 .form-field {
   &-outer {
     height: 45px;
-    background-color: $light-grey  !important;
+    background-color: $grey-light  !important;
```
Comment (helenb) [inline mresponse/frontend/app/src/components/input-field/field.scss]: is the !important definitely needed?

File: mresponse/frontend/app/src/components/textarea/textarea.scss [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+      width: calc(100% - 20px);
+      padding: 10px;
+      background-color: $off-white;
+      border-radius: 3px;
+      border: 0.5px solid $border-color;
+      font-size: 14px;
+      line-height: 1.6;
+
+      &::placeholder {
+          color: rgba(#38383d, 0.15);
```
Comment (helenb) [inline mresponse/frontend/app/src/components/textarea/textarea.scss]: could a colour variable be used here?

File: mresponse/frontend/app/src/components/input-field/field.scss (hunk 4 above)
Comment (thisoldbear) [reply mresponse/frontend/app/src/components/input-field/field.scss]: I didn't check. Will take a look.

Comment (thisoldbear) [reply mresponse/frontend/app/src/components/input-field/field.scss]: Seems there are a few `!important` in the styles 😬

File: mresponse/frontend/app/src/components/textarea/textarea.scss (hunk 5 above)
Comment (thisoldbear) [reply mresponse/frontend/app/src/components/textarea/textarea.scss]: Added 👍

**Pull request merged**